src/test/rpc_tests.cpp

```diff
@@ -62,7 +62,7 @@ BOOST_FIXTURE_TEST_CASE(rpc_addmultisig, TestNetFixture)
     string short1(address1Hex, address1Hex+sizeof(address1Hex)-2); // last byte missing
```

Comment:
Interesting that we remove 2 here but only add 1 below...

Main problem with these tests, they check if something is thrown but have no valid way to check what (this is an issue with a lot of the tests actually).